README.md

```diff
@@ -1 +1,69 @@
-# budget_app
\ No newline at end of file
+# Budget app
+
+Budget app is about building a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what.
+
+## Built With
+
+- Ruby -v 3.0.2
+- Rails -v 7.0.4
+- gem 'rspec'
+- gem 'sass-rails'
+
+### Presentation
+
+[Video](https://www.loom.com/share/711cc33bc1644273b2a30b881b96c54d)
+
+### Live 
+
+[budgetapp](https://budget-app-arnaud.herokuapp.com/)
+
+### Getting Started
+
+To get a local copy up and running follow these simple steps:
+
+#### Prerequisites
+
+- Ruby -v 3.0.2
+- Rails -v 7.0.4
+- Postgresql
+
+#### Setup
+
+- Open the terminal and run the following commands:
+
+```
+- git clone git@github.com:ArnaudBand/budget_app.git
+- rails new . --database=postgresql -T
+- cd budget_app
+
+```
+
+- Run the application on the server using this command
+
+`rails s`
+
+#### Author
+
+*Bandonkeye Arnaud*
+
+- Github: [ArnaudBand](https://github.com/ArnaudBand)
+- Linkedin: [Arnaud](https://www.linkedin.com/in/ArnaudBandonkeye/)
+- Twitter: [@ba104781](https://twitter.com/ba104781)
+
+#### Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/ArnaudBand/budget_app/issues).
+
+#### Show your support
+
+Give a star if you like this project!
+
+#### Acknowledgments
+
```

Comment:
- [ ] It's a nice and smart thing to give appropriate credit to **[Gregoire Vella on Behance](https://www.behance.net/gregoirevella)**, the author of [the original design](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), as required by the [Creative Commons License](https://creativecommons.org/licenses/). Kindly ensure to add this under the **Acknowledgements** section of your ReadMe.